examples/mobile/UIComponents/components/view-controls/dropdownlist/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../lib/tau/mobile/js/tau.js">
```

Comment:
Is it correct location for the tau.css, style.css, and tau.js.

Reply:
I see but I think they are not needed at all since tau loads subpages into main html to separate ui-page. I will remove them.

Reply:
Removing them may cause issue for tizen.org where we loads subpages directly. I will fix the paths. Thanks.